waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -48,7 +45,9 @@ migrateDev projectDir migrateArgs = do
           else -- NOTE(martin): On Linux, command that `script` should execute is treated as one argument.
             ["-feqc", unwords prismaMigrateCmd, "/dev/null"]
 
-  runNodeCommandAsJob serverDir "script" scriptArgs J.Db
+  -- TODO: Prisma migrate will generate a client whether we want it or not. Therefore, I passed
+  -- in the server's client location. We should probably generate one for the web app as well.
+  runNodeCommandAsJobWithExtraEnv [serverPrismaClientOutputDirEnv] serverDir "script" scriptArgs J.Db
```

Comment:
Advice welcome :) @Martinsos @shayneczyzewski

Reply:
Do we want to possibly skip client generation when running migrate? I wonder if that may be confusing to users though if they expect it to. But, it appears you _can_ skip that step on migrate if you wanted: https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-dev Not sure what other generators `--skip-generate` will skip though. But this way, we could differentiate when we generate clients for them.

But yeah, this feels like we are opening a can of worms having to detect under what other instances clients could be generated out of sync. Maybe we just force client generation for the web app after migrating in the server context?

Reply:
Did this in the end: https://github.com/wasp-lang/wasp/pull/962#discussion_r1102909870